wordpress-mobile/WordPressApi
# Updated the Example project to CocoaPods 1.0

Updates the Example project to use CocoaPods 1.0.

To test:
1. Check the changes.
2. Install the Pods and make sure the lock file is unchanged.
3. Run the unit tests.

Needs review: @aerych
